.github/workflows/remove-needs-ok-to-test.yaml

```diff
@@ -0,0 +1,16 @@
+name: Remove needs-ok-to-test
+
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+    types: [labeled]
+
+jobs:
+  remove-label:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Checkout shouldn't be needed here... Or is it? There's no file action on the repo...

Reply:
After testing It doesn't seem to be... Previously I followed example in action repo, so I assumed that it is. Fixed :+1: